cvat-ui/src/actions/formats-actions.ts

```diff
@@ -0,0 +1,43 @@
+import { AnyAction, Dispatch, ActionCreator } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+
+import getCore from '../core';
+
+const cvat = getCore();
+
+export enum FormatsActionTypes {
+    GETTING_FORMATS_SUCCESS = 'GETTING_FORMATS_SUCCESS',
+    GETTING_FORMATS_FAILED = 'GETTING_FORMATS_FAILED',
+}
+
+export function gettingFormatsSuccess(formats: any): AnyAction {
```

Comment:
I would prefer to have names like "onSuccessServerFormats" and "OnFailureServerFormats".

Reply:
I believe ServerFormats isn't the best choice.
AnnotationsFormats or shorter variant just Formats.